src/constants/tokens.ts

```diff
@@ -1114,3 +1114,15 @@ export function getTokens(chainId: ChainId): Token[] {
   }
   return vaults;
 }
+
+export function getOneTokens(chainId: ChainId): Token[] {
+  const names = Object.keys(TOKENS);
+  const vaults: Token[] = [];
+  for (let name of names) {
+    if (chainId in TOKENS[name] && TOKENS[name][chainId].isOneToken) {
+      vaults.push(TOKENS[name][chainId]);
+    }
+  }
+  return vaults;
+}
```

Comment:
You'd want to test this, but you could compact all that to really one line:

```
  const vaults: Token[] = Object.values(TOKENS)
    .map((t) => t[chainId])
    .filter((t) => t?.isOneToken)
    .filter((t) => t) as Token[];
```

Not sure if that is superior or not as it reduces readability, just a suggestion.